bin.ts

```diff
@@ -196,6 +206,8 @@ Options:
   --release           Set the date of the specified release
   --create            Create a new release
 
+  --sortReleases      Sort releases (default: true)
```

Comment:
For consistency with other options I'd use kebab-case style for the name `sort-releases` instead of `sortReleases`.

And for consistency with `--no-v-prefix`, I'd rename it to `--no-sort-releases`, because the default behavior is to sort them and the user only can disable it.